### PR TITLE
Add clone_repo usage

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
CVS-157301

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

File: .github/workflows/treon_precommit.yml [hunk 1]
```diff
@@ -27,15 +27,15 @@ on:
...
+  cancel-in-progress: false
 
 permissions:
   contents: read
 
 jobs:
   build_treon:
     strategy:
-      fail-fast: true
+      fail-fast: false
```
Comment (eaidova) [inline .github/workflows/treon_precommit.yml]: do not forget to return back

File: notebooks/mobilevlm-language-assistant/mobilevlm-language-assistant.ipynb [hunk 2]
```diff
@@ -72,18 +72,36 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```
Comment (eaidova) [inline notebooks/mobilevlm-language-assistant/mobilevlm-language-assistant.ipynb:75]: could you please add accelerate to installed packages according to recommendation from github actions

Comment (aleksandr-mokrov) [reply notebooks/mobilevlm-language-assistant/mobilevlm-language-assistant.ipynb:75]: accelerate version is updated

File: .github/workflows/treon_precommit.yml (hunk 1 above)
Comment (aleksandr-mokrov) [reply .github/workflows/treon_precommit.yml]: returned

**Pull request merged**